README.md

```diff
@@ -54,8 +54,9 @@ Usage of Shield requires the following:
 ### Installation
 
 Installation is done through Composer.
-
-    > composer require codeigniter4/shield
+```console
```

Comment:
`console` is for shell output, `shell` is for commands: https://github.com/github/linguist/blob/master/lib/linguist/languages.yml

Reply:
In my understanding, `console` is shell session. 
All characters including command prompts and commands and outputs in the terminal.

`shell` is a shell script that is program code written by shell.

Reply:
That's right. Console is actually an alias for ShellSession:
```yml
ShellSession:
  type: programming
  extensions:
  - ".sh-session"
  aliases:
  - bash session
  - console
```

@kenjis Are you saying I have misinterpreted how this should be used?

Reply:
I think if we copy and paste the terminal output, we use `console`.
If it is shell script, we use `shell`.

So I am Okay with this PR.